src/main/java/newbank/server/NewBankClientHandler.java

```diff
@@ -44,22 +46,31 @@ private void initialiseSupportedCommands() {
     commands.put("QUIT", QuitCommand::new);
     commands.put("REGISTER", RegisterCommand::new);
     commands.put("SHOWMYACCOUNTS", ShowAccountsCommand::new);
+    commands.put("MOVE", MoveMoneyCommand::new);
     commands.put("DEFAULT", DefaultCommand::new);
     commands.put("UNKNOWN", UnknownCommand::new);
   }
 
   private Command getCommand(final String name, final String[] tokens) {
     return commands.getOrDefault(name, UnknownCommand::new).makeCommand(bank, tokens, customer);
   }
-
   private boolean processRequest(final String request) {
     final String[] tokens = request.trim().split("\\s+");
 
     assert (tokens.length > 0);
 
     final String commandName = tokens[0].toUpperCase();
+    final Command command = getCommand(commandName, tokens);
 
-    out.println(getCommand(commandName, tokens).execute());
+    if (tokens.length >= 2 && tokens[1].equalsIgnoreCase("HELP")) {
+      out.println(command.getUsage());
+    } else {
+      try {
+        out.println(command.execute());
+      } catch(CommandInvalidSyntaxException e) {
+        out.println(command.getUsageInvalidSyntax());
```

Comment:
The error message should be built here, with help from the getUsage() method I would say.

Regards,
Jules

Reply:
Ahh, I see. Can give it a try :+1: